spec/features/admin_deletes_review_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+feature 'admin deletes article', %Q{
+  As an admin
+  I want to delete an article
+  So that I can filter out unwanted articles
+} do
+
+  let!(:admin) { FactoryGirl.create(:admin_user) }
+  let!(:review) { FactoryGirl.create(:review) }
+
+  scenario 'admin deletes a review' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.